src/index.md

```diff
@@ -0,0 +1,5 @@
+---
+name: 'main'
+---
+
+# Hello World!
```

Comment:
Update this documentation so that it's actually useful.

Also add a `design-tokens/index.md` page with links.